config.schema.yaml

```diff
@@ -0,0 +1,40 @@
+---
+credentials:
+  description: "Array of credentials for cloud providers"
+  required: true
+  type: "array"
```

Comment:
This should be `object` since it's a dict.

You will also need to use references to get this to work - see vsphere pack for example of that https://github.com/StackStorm-Exchange/stackstorm-vsphere/blob/master/config.schema.yaml#L7